extensions/amp-font/0.1/amp-font.js

```diff
@@ -174,12 +165,12 @@ export class AmpFont extends AMP.BaseElement {
    * @private
    */
   onFontLoadFinish_(addClassName, removeClassName) {
+    const body = this.getAmpDoc().getBody();
     if (addClassName) {
-      this.documentElement_.classList.add(addClassName);
+      body.classList.add(addClassName);
```

Comment:
as discussed offline, let's also add it to `htmlElement`, in addition to the need for this backward compatible, `fixedLayer` elements get their own `body` so the fallback font's will be for `position:fixed` elements in iOS.

Reply:
+1